packages/Uploader/src/components/ProgressBar/ProgressBar.styles.js

```diff
@@ -0,0 +1,27 @@
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+
+export default {
```

Comment:
Interesting pattern. Kind of trivial , but I like `styles.container` better than `containerStyles`.  

Something worth striving for consistently, or just an author's choice kind of thing?

Reply:
lol, this is the pattern that I follow on my side project. 
I guess, I implemented without noticing, to me is slightly better and easier to read and easier to import because you only have to import the styles object.